sdk/core/core-auth/review/core-auth.api.md

```diff
@@ -11,6 +11,7 @@ export interface AccessToken {
     expiresOnTimestamp: number;
     refreshAfterTimestamp?: number;
     token: string;
+    tokenType: string;
```

Comment:
Sometimes people implement their own version of `TokenCredential` outside of Identity, for a number of reasons, e.g. maybe for mock testing (like I think we do this in test credential):

```ts
const mockCredential: TokenCredential = {
  async getToken() {
    return { token: "aaaa", expiresOnTimestamp: ((new Date().getTime()) / 1000) + 10000 };
  }
}
```

This will be a breaking change for this scenario if this new property is required. Maybe we need a major version bump here? Alternatively, would it be possible to make this property optional (and have it default to "Bearer" when it is not set)?

Reply:
I like this being optional

Reply:
made it optional and added a helper to deduce the type by default, since we can't set a default type